packages/thirdweb/src/wallets/private-key.ts

```diff
@@ -57,12 +64,65 @@ export type PrivateKeyAccountOptions = {
  * ```
  * @wallet
  */
-export function privateKeyAccount(options: PrivateKeyAccountOptions): Account {
-  if (!options.privateKey.startsWith("0x")) {
-    options.privateKey = "0x" + options.privateKey;
-  }
-  const viemAccount = privateKeyToAccount(options.privateKey as Hex);
-  return viemToThirdwebAccount(viemAccount, options.client);
+export function privateKeyAccount(
+  options: PrivateKeyAccountOptions,
+): LocalAccount {
```

Comment:
unless we absolutely *have* to have different types for different types of accounts I'd love to be able to avoid this here